packages/listbox/test-suites/ListboxMixin.suite.js

```diff
@@ -383,6 +383,52 @@ export function runListboxMixinSuite(customConfig = {}) {
         expect(el.modelValue).to.equal(20);
       });
 
+      it('should reset selected value to initial value', async () => {
+        const el = await fixture(html`
+          <${tag}>
+            <lion-options slot="input">
+              <${optionTag} .choiceValue=${10}>Item 1</${optionTag}>
+              <${optionTag} .choiceValue=${20} checked>Item 2</${optionTag}>
+            </lion-options>
+          </${tag}>
+        `);
+        el.setCheckedIndex(0);
+        expect(el.modelValue).to.equal(10);
+        el.reset();
+        expect(el.modelValue).to.equal(20);
+      });
+
+      it('should reset selected value to "Please select account" with has-no-default-selected attribute', async () => {
+        const el = await fixture(html`
+          <${tag} has-no-default-selected>
+            <lion-options slot="input">
+              <${optionTag} .choiceValue=${10}>Item 1</${optionTag}>
+              <${optionTag} .choiceValue=${20}>Item 2</${optionTag}>
+            </lion-options>
+          </${tag}>
+        `);
+        el.setCheckedIndex(1);
+        expect(el.modelValue).to.equal(20);
+        el.reset();
+        expect(el.modelValue).to.equal('');
+      });
+
+      it('should reset selected value to initial values when multiple-choice', async () => {
+        const el = await fixture(html`
+          <${tag} multiple-choice has-no-default-selected>
```

Comment:
Not fully happy to have to add `has-no-default-selected`, but that is not part of this pull request